### ISSUE TITLE
tester l'inscription avec une adresse hotmail

### ISSUE THREAD

**Issue by emmrichard** [axel584/ikurso#423]
Voir si les messages arrivent dans les spam, pour hotmail. com et hotmail.fr

**Comment by axel584** [comment]
@emmrichard : ça me semble bon celle ci, car on a des élèves qui s'inscrivent avec hotmail ?

**Comment by emmrichard** [comment]
@axel584 : oui, on a pas mal d'élèves, dont une partie ont complété leur fiche donc ils ont reçu le mail.

**Comment by axel584** [comment]
@emmrichard : le problème semble être les leçons car elles ne sont pas envoyées avec ikurso@esperanto-france.org comme expéditeur et elles ne sont pas envoyés par Amazon SES (d'ailleurs, on ne peut pas envoyer via amazon SES un message avec une autre adresse d'expéditeur que ikurso)

**Comment by emmrichard** [comment]
J'ai fait un test d'inscription hier soir en utilisant mon adresse @
hotmail.fr, je n'ai rien reçu, j'ai redemandé l'envoi du mail. Ce matin,
toujours rien. J'ai retenté l'envoi du mail d'activation mais rien à faire.
